docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md

```diff
@@ -247,13 +247,31 @@ If the key is protected by a passphrase in the PKCS12 archive, you are prompted
 
 #### Verify certificate chain
 
-To verify the certificate chain, execute the following command:
+Perform the following steps to verify your certificates:
 
-```sh
-openssl verify -CAfile bundle.pem cert.pem
-```
+1. Execute the following verify command which checks the database node certificate "node.crt" against the root CA certificate "ca.crt":
+
+    ```sh
+    openssl verify -CAfile bundle.pem cert.pem
+    ```
+
+    The `bundle.pem` file is a certificate bundle containing the root certificate and any intermediate certificates in the PEM format.
+
+1. Verify that the node certificate (`node.ip_address.crt`) and the node private key (`node.ip_address.key`) match. See [How do I verify that a private key matches a certificate?](https://www.ssl247.com/knowledge-base/detail/how-do-i-verify-that-a-private-key-matches-a-certificate-openssl-1527076112539/ka03l0000015hscaay/)
 
-The `bundle.pem` file is a certificate bundle containing the root certificate and any intermediate certificates in the PEM format.
+1. Verify that the node certificate expiration is at least 3 months by checking the validity field in the output of the following command:
+
+    ```sh
+    openssl x509 -in node.crt -text -noout
+    ```
+
+1. Verify that the node certificate Common Name or Subject Alternate Name contains the IP address or DNS name of each on-prem node on which the nodes are deployed.
```

Comment:
It would be good to clarify one issue here that customers run into -

the CN/SAN needs to have the same entry as the node's entry in the onprem provider. If the node was added as a DNS address in the onprem provider, the same DNS entry should be present here. 

I'm not sure how to convey this point exactly. Customers sometimes enter a DNS address in the onprem provider but add a cert with the resolved IP address.